src/sdk/namespace_fs.js

```diff
@@ -1170,7 +1172,8 @@ class NamespaceFS {
         const fs_context = this.prepare_fs_context(object_sdk);
         const open_mode = 'w';
         try {
-            const MD5Async = config.NSFS_CALCULATE_MD5 ? new (nb_native().crypto.MD5Async)() : undefined;
+            const md5_enabled = params.force_md5_etag;
+            const MD5Async = md5_enabled ? new (nb_native().crypto.MD5Async)() : undefined;
```

Comment:
Following up on @utkarsh-pro comment [here](https://github.com/noobaa/noobaa-core/pull/7297#discussion_r1191013617) on the account PR, maybe we should also keep the `config.NSFS_CALCULATE_MD5` as a global toggle. by default it should stay false, but it will allow us to turn it on globally when needed